src/app/redux/UserSaga.js

```diff
@@ -418,13 +499,32 @@ function* promptTosAcceptance(username) {
 
 function* getFeatureFlags(username, posting_private) {
     try {
-        const flags = yield call(
-            [api, api.signedCallAsync],
-            'conveyor.get_feature_flags',
-            { account: username },
-            username,
-            posting_private
-        );
+        let flags;
+        if (!posting_private && hasCompatibleKeychain()) {
+            flags = yield new Promise((resolve, reject) => {
+                window.steem_keychain.requestSignedCall(
+                    username,
+                    'conveyor.get_feature_flags',
```

Comment:
@roadscape FYI-- here's where feature flag fetch is, and I believe you should see it in testing since the window pops up twice, one for this call.